.travis.yml

```diff
@@ -13,18 +13,15 @@ env:
   matrix:
     - PGVERSION=9.5
     - PGVERSION=9.6
+    - PGVERSION=10
 before_install:
-  - git clone -b v0.6.1 --depth 1 https://github.com/citusdata/tools.git
+  - git clone -b pg10_hack --depth 1 https://github.com/citusdata/tools.git
   - sudo make -C tools install
   - setup_apt
-  - curl https://install.citusdata.com/community/deb.sh | sudo bash
```

Comment:
No, it only sets up `pgdg` for `9.6`, and only if it's not already present, as it's meant to be the easiest user experience for new Citus users. If users need `9.5` or something, they need to set that up themselves.

`setup_apt` is the script which has always set up our Travis box to know about PGDG; it predates `install.citusdata.com` by quite a bit. It's been augmented to add the `10 main` repo.